tasks/95_pairwise/pairwise.js

```diff
@@ -0,0 +1,12 @@
+export function pairwise(arr, num) {
+  let arrToSearch = arr;
+  return arr.reduce((sum, item, index) => {
+    for (let i = index + 1; i < arr.length; i++) {
+      if (item + arr[i] === num) {
+        arrToSearch[i] = '';
```

Comment:
here `arrToSearch[i] === arr[i]`
and `arrToSearch === arr`. It's the same object in memory

The method what calls `pairwise` does not expect that input array would be changed